src/domains/wallet/components/WalletsGroup/WalletsGroup.blocks.tsx

```diff
@@ -153,6 +142,8 @@ export const GroupNetworkTotal: React.VFC<WalletsGroupNetworkTotalProperties> =
 			return <GroupSkeleton width={140} className="h-5" innerClassName="h-4" />;
 		}
 
+		const totalNetworkBalance = wallets.reduce((balance, wallet) => balance + wallet.balance(), 0);
```

Comment:
@alfonsobries do you think we could use BigNumber to avoid any floating point arithmetic overflow :hmhmhm: 

```suggestion
		const totalNetworkBalance = wallets.reduce((balance, wallet) => balance.plus(wallet.balance()), BigNumber.ZERO);

```